images/sdn/Dockerfile.rhel

```diff
@@ -21,6 +21,9 @@ COPY --from=rhel8-builder /go/src/github.com/openshift/sdn/host-local /usr/bin/c
 RUN mkdir -p /opt/cni/bin/rhel7
 COPY --from=rhel7-builder /go/src/github.com/openshift/sdn/sdn-cni-plugin /opt/cni/bin/rhel7/openshift-sdn
 
+FROM registry.ci.openshift.org/ocp/4.8:cli AS cli
```

Comment:
Move `FROM registry.ci.openshift.org/ocp/4.8:cli AS cli` above `FROM registry.ci.openshift.org/ocp/4.8:base` else you are copying the binary onto itself.